### DISCUSSION TITLE
Possibilité de pouvoir kick un membre lors de l'ajout d'un nouveau en étant ainé d'une guilde.

### DISCUSSION THREAD

**Discussion by AlexisLhotte** [Crownicles/Crownicles#1208] [Suggestions fermées]
Le petit truc que j'ai remarqué, c'est que lorsqu'on dois, lors de l'ajout d'un membre, exclure un autre lorsque la guilde est pleine.

Hors seul le chef de guilde peut le faire, donc si on compare le chef de guilde à un administrateur d'un serveur, l'ainé devrais être comme un "modérateur" et pouvant bénéficier de cette fonction uniquement lors de l'ajout d'un nouveau membre

**Comment by Pietagorh** [comment]
Pas d'accord
Le problème est que c'est plus compliqué de faire revenir quelqu'un que l'ainé a kick (genre par erreur) que de kick quelqu'un ajouté sans l'accord des autres
Honnêtement j'préfère que l'elder puisse pas kick

**Comment by AlexisLhotte** [comment]
Après l'elder à été choisi par le chef de guilde, donc par sa confiance, donc une option accordé au chef de guilde lui donnant la possibilité de donner cette permission ou de la retirer à l'elder, si dans le cas ou c'est le chef de guilde qui donne l'accord, il ne devrait pas y avoir de souci

**Comment by Cyyrene** [comment]
oui, l'elder devrait pouvoir exclure des gens de la guilde.

**Comment by Procion06** [comment]
c'est une mauvaise idée
si ton chef de guilde n'est pas capable de gérer le renvoi/ajout de joueurs, change vite de guilde
